plugins/modules/rulebook_activation.py

```diff
@@ -118,6 +124,7 @@ def main():
         extra_vars=dict(type="dict"),
         enabled=dict(type="bool", default="true"),
         state=dict(choices=["present", "absent", "restarted"], default="present"),
+        awx_token=dict(no_log=True),
```

Comment:
Just added in the `no_log=True` parameter to the `awx_token` option as sanity complained about it (correctly).